### DISCUSSION TITLE
giscus-docusaurus | WenjieWei Blog

### DISCUSSION THREAD

**Discussion by giscus** [wjwei-handsome/wjwei-docusaurus#8] [General]
# docs/Potpourri/giscus-docusaurus

简单介绍如何在 Docusaurus 上配置 Giscus 评论系统，并解决更新后的 bug

https://www.wjwei.blog/blog/giscus-docusaurus

**Comment by dbxxx-ac** [comment]
感谢，v3.0 亲测可用
